install.md

```diff
@@ -1,5 +1,27 @@
 # libpod Installation Instructions
 
+## Installing packaged versions of Podman
+
+#### Fedora, Centos, OpenSUSE, SUSE:
+
+yum -y install podman
+
+#### RHEL7
+
+Enable the extras repo.
+yum -y install podman
+
+#### RHEL8
+
+yum -y install podman
+
+#### Ubuntu and friends
+...
+
+#### Gentoo - Read Building from scratch
+
```

Comment:
It's now in the portage tree:
```
# ACCEPT_KEYWORDS='~amd64' emerge app-emulation/libpod 
```

Reply:
Do you have a description on how to install it?

Reply:
That's it - that's the incantation for installing it. `ACCEPT_KEYWORDS` is because it and cri-o and something else aren't stable; that allows non-stable packages to be install. `emerge` is the command that fetches compiles and installs.

Reply:
On second thought it might be better to just write:
```
# emerge app-emulation/libpod
```
If run today, it will fail but give instructions on how to proceed, in a way that is familiar to all Gentoo users. Some future day, though, libpod will be marked stable and the `~amd64` will not be necessary. Best to keep it all simple for now.